packages/modules/vehicles/tronity/api.py

```diff
@@ -22,7 +22,7 @@ def fetch_soc(config: TronityVehicleSocConfiguration, soc_update_data: SocUpdate
 
 
 def is_token_valid(access_token: str) -> bool:
-    if not access_token:
+    if not access_token or access_token == 'None':
```

Comment:
Ist das `access_token` in diesem Fall wirklich ein String "None", oder `None`? Bei letzterem wäre die korrekte Prüfung:
```suggestion
    if not access_token or access_token is None:
```

Reply:
Es ist tatsächlich 'None'. Im Broker ist ein null Wert (""), hier im Code kommt dann 'None' an. So gesehen past die Prüfung.